lib/data/wrapper.js

```diff
@@ -153,11 +153,16 @@ const data = {
 
     // It would be preferable to have an sproxyd batch delete route to
     // replace this
-    batchDelete: (locations, log) => {
+    batchDelete: (locations, log, noDelete) => {
         // TODO: The method of persistence of sproxy delete key will
         // be finalized; refer Issue #312 for the discussion. In the
         // meantime, we at least log the location of the data we are
         // about to delete before attempting its deletion.
+        const skipIfBackend = ['aws-test'];
+        if (noDelete &&
+        skipIfBackend.indexOf(locations[0].dataStoreName) > -1) {
+            return;
```

Comment:
That's going to get changed. It's just gonna take a little more finagling to get the clientType :)